catalyst_voices/packages/internal/catalyst_voices_services/lib/src/crypto/key_derivation.dart

```diff
@@ -3,6 +3,10 @@ import 'package:catalyst_voices_models/catalyst_voices_models.dart';
 
 /// Derives key pairs from a seed phrase.
 final class KeyDerivation {
+  /// See: https://github.com/input-output-hk/catalyst-voices/pull/1300
+  static const int _purpose = 508;
+  static const int _type = 139;
+
```

Comment:
Just for alignment with the spec,  I suggest adding this.
```suggestion
static const int _account = 0; // Future Use
```